bin/sce_qc_report.R

```diff
@@ -152,12 +159,16 @@ metadata_list <- list(
   has_cellhash = has_cellhash,
   filtered_cells = ncol(filtered_sce),
   unfiltered_cells = ncol(unfiltered_sce),
-  filtering_method = filtered_sce_meta$filtering_method,
+  empty_drops_filtering_method = filtered_sce_meta$filtering_method,
```

Comment:
Do we use this downstream at all? If not yet, then a small change:
```suggestion
  droplet_filtering_method = filtered_sce_meta$filtering_method,
```

Reply:
No we don't, so I'll change this.